### DISCUSSION TITLE
Select/text field

### DISCUSSION THREAD

**Discussion by csikoszoltan** [filamentphp/filament#395] [Help] [answered]
Hey!

I'm trying to use the select field like if the typed option doesn't exist, it should make it in the database.
It's like a select/text field combo.
Is it possible to do this somehow or should it be a custom component?

Thanks!

**Comment by danharrin** [comment, marked as answer]
Hey! This should be a custom component.

**Reply by csikoszoltan** [reply]
Ok, thanks!
